skipchain/api.go

```diff
@@ -23,6 +23,9 @@ const (
 	ErrorVerification
 	// ErrorOnet indicates an error from the onet framework
 	ErrorOnet
+	// ErrorBlockInProgress indicates that currently a block is being formed
+	// and propagated
+	ErrorBlockInProgress
```

Comment:
Also indicates somewhere that change of protocol that only the leader can initiate requests for a new block to be signed, that's why this error exists in the first place too no ?